ttyd-tools/rel/include/evt_cmd.h

```diff
@@ -298,6 +293,13 @@ using evt_helper_int_array = int32_t[];
 	), \
 	reinterpret_cast<int32_t>(function), \
 	##__VA_ARGS__ ,
+    
+// User function calls with unchecked parameter counts.
+// (Alternative to using the EVT_DECLARE_USER_FUNC macro w/-1)
+#define UNCHECKED_USER_FUNC(function, ...) \
```

Comment:
This is definitely a good thing to have, but I'm unsure about the naming. Specifically:
* For readability purposes I feel like it should start with `USER_FUNC_...` to match the standard command.
* Not sure about calling it "unchecked", it feels a bit long. Perhaps `USER_FUNC_UNSAFE`?

Reply:
Sounds good, changed.